R/check_description.R

```diff
@@ -1,4 +1,26 @@
-#' Check the DESCRIPTION file
+#' Check the `DESCRIPTION` file
+#'
+#' The `DESCRIPTION` file contains the most important meta-data of the package.
+#' A good `DESCRIPTION` is tidy, has a meaningful version number, full
+#' author details and a clear license.
+#'
+#' #' This function ensures the `DESCRIPTION` is tidy, using `tidy_desc()`.
+#'
+#' The version number of the package must have either a `0.0` or a `0.0.0`
+#' format (see this [discussion](https://github.com/inbo/checklist/issues/1) why
```

Comment:
Just a suggestion, seems more logical to me.

```suggestion
#' format (see [this discussion](https://github.com/inbo/checklist/issues/1) why
```